<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:#B9B4C7</string>
   </property>
   <widget class="QLabel" name="labelMain">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>320</width>
      <height>59</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>186</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>500</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#5C5470</string>
    </property>
    <property name="text">
     <string>     CATALOGUE OF COMPRESSORS</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>279</width>
      <height>99</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>58</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>500</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelDescrp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>331</width>
      <height>171</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>58</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>400</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#5C5470</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labelBrand">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>57</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:</string>
         </property>
         <property name="text">
          <string>BRAND</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelModel">
         <property name="minimumSize">
          <size>
           <width>46</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>57</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:</string>
         </property>
         <property name="text">
          <string>MODEL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPrice">
         <property name="minimumSize">
          <size>
           <width>37</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>57</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:</string>
         </property>
         <property name="text">
          <string>PRICE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelGetBrand">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>129</width>
           <height>57</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelGetModel">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>129</width>
           <height>57</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelGetPrice">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>129</width>
           <height>57</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushFET">
       <property name="minimumSize">
        <size>
         <width>53</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#352F44</string>
       </property>
       <property name="text">
        <string>FET</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushTUBE">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#352F44</string>
       </property>
       <property name="text">
        <string>TUBE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushVCA">
       <property name="minimumSize">
        <size>
         <width>57</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#352F44</string>
       </property>
       <property name="text">
        <string>VCA</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
